src/providers/VoiceFocusProvider/docs/VoiceFocusProvider.stories.mdx

```diff
@@ -21,6 +24,16 @@ This provider is independent from `MeetingProvider`. You can put `VoiceFocusProv
 
 You can access the state by using the [useVoiceFocus](/docs/sdk-hooks-usevoicefocus--page) hook.
 
+## Props
+
+<Props of={VoiceFocusProvider} />
+
+You can set a appropriate module complexity in `VoiceFocusProvider`. Amazon Voice Focus related logs can be found in the dev tool when the feature is enabled.
```

Comment:
I think you dont have to explicilty write the props here, the `<Props of ={VoiceFocusProvider} />` will show the props table. Just check if you have the correct documentation in the props itself in `VoiceFocusProvider`.

Reply:
I guess I can remove the sentence "You can set a appropriate module complexity in `VoiceFocusProvider`." ?

Reply:
Yeah, but should we give some examples saying if VF is supported you should see "..." log, similarly for if enabled successfully or failed or not supported.

Add few helpers logs to let builders know what to look when we have the VF enabled and working. Also, if we do not have sufficient logging, suggest to add with this PR itself:
- When VF is disabled.
- When it is enabled.
- Supported / Not supported cases.
- Failure cases.
- Switch between a normal audio input device and the VF audio transform device.

Reply:
Good point! I plan to add logs in the VF component and also the corresponding documentation. 

But this PR is for correcting VF documentation, I don't want to increase the scope. You can expect my next PR including sufficient logging ^^